schemas/input/csl-data.json

```diff
@@ -4,425 +4,477 @@
+          "type": "string"
+        },
+        "issue": {
+          "type": ["string", "number"]
+        },
+        "jurisdiction": {
+          "type": "string"
+        },
+        "keyword": {
+          "type": "string"
+        },
+        "locator": {
+          "type": ["string", "number"]
+        },
+        "medium": {
+          "type": "string"
+        },
+        "note": {
+          "type": "string"
+        },
+        "number": {
+          "type": ["string", "number"]
+        },
+        "number-of-pages": {
+          "type": ["string", "number"]
+        },
+        "number-of-volumes": {
+          "type": ["string", "number"]
+        },
+        "original": {
+          "title": "Original - Related Item",
+          "description": "Used to define metadata for an original publication.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "original-publisher": {
+          "description": "[Deprecated - Use `publisher` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-publisher-place": {
+          "description": "[Deprecated - Use `publisher-place` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-title": {
+          "description": "[Deprecated - Use `title` on the `original` related item instead]",
+          "type": "string"
+        },
+        "page": {
+          "type": ["string", "number"]
+        },
+        "page-first": {
+          "type": ["string", "number"]
+        },
+        "part": {
+          "type": ["string", "number"]
+        },
+        "part-title": {
+          "type": "string"
+        },
+        "PMCID": {
+          "type": "string"
+        },
+        "PMID": {
+          "type": "string"
+        },
+        "printing": {
+          "type": ["string", "number"]
+        },
+        "publisher": {
+          "type": "string"
+        },
+        "publisher-place": {
+          "type": "string"
+        },
+        "references": {
+          "type": "string"
+        },
+        "reviewed": {
+          "title": "Reviewed - Related Item",
+          "description": "In a review, defines metadata for the reviewed item.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "reviewed-title": {
+          "description": "[Deprecated - Use `title` on the `reviewed` related item instead]",
+          "type": "string"
+        },
+        "scale": {
+          "type": "string"
+        },
+        "section": {
+          "type": "string"
+        },
+        "source": {
+          "type": "string"
+        },
+        "status": {
+          "type": "string"
+        },
+        "supplement": {
+          "type": ["string", "number"]
+        },
+        "title": {
+          "type": "string"
+        },
+        "title-short": {
+          "type": "string"
+        },
+        "translated-title": {
+          "description": "[Deprecated - Q: what TODO?]",
```

Comment:
```suggestion
```

This one `translated-*` variable will suffice for the vast vast majority of cases. I don't think standard CSL needs to include `translated` as a related element (though this structure provides a great hook for making the multilingual parts of CSLm/Jurism a separate module).